tests/test-requirement.txt

```diff
@@ -3,3 +3,5 @@ pytest
 pytest-cov
 pytest-mock
 requests-mock
+types-requests
```

Comment:
I would move them to types-requirements.

Reply:
The practice in most agents right now is having them in `test-requirements.txt`.

Reply:
That's the new practice.